app/controllers/api/v1/court_hearings_controller.rb

```diff
@@ -41,9 +42,9 @@ def move
       end
 
       def should_save_in_nomis?
-        do_not_save_to_nomis = params['do_not_save_to_nomis'] == 'true'
+        save_to_nomis = params.fetch('do_not_save_to_nomis', 'false') != 'true'
```

Comment:
just a sanity check: is the parameter name `do_not_save_to_nomis` or `save_to_nomis` ?

Reply:
The query param is definitely `do_not_save_to_nomis`. I'm making the bit where we evaluate the business logic clearer as it was a bit garbled before